caddyhttp/httpserver/server.go

```diff
@@ -562,8 +562,11 @@ func (ln tcpKeepAliveListener) Accept() (c net.Conn, err error) {
 	if err = tc.SetKeepAlive(true); err != nil {
 		return
 	}
-	if err = tc.SetKeepAlivePeriod(3 * time.Minute); err != nil {
-		return
+	// OpenBSD has no user-settable per-socket TCP keepalive
```

Comment:
Can you link to the GitHub issue and/or PR so that it is easy for future readers to find out more about this?

Reply:
done